packages/api/src/models/showedSpotlightModel.js

```diff
@@ -70,6 +70,7 @@ class ShowedSpotlight extends Model {
         sensor_reading_chart_end: { type: ['string', 'null'] },
         repeat_management_plan_creation: { type: 'boolean' },
         repeat_management_plan_creation_end: { type: ['string', 'null'] },
+        manage_custom_expense_type: { type: ['string', 'null'] },
```

Comment:
I think the type should be `boolean`, and `manage_custom_expense_type_end` needs to be added with `{ type: ['string', 'null'] }`.
The API is returning 400, but this should fix it!

Reply:
Great catch! Updated this now, should be fixed 🙂